docker/bootstrap_internal.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+export CUSTOMSETTINGS="docker.localsettings-docker"
```

Comment:
This works because its using `importlib`. Its OK not to have it normally
importable since it's only used to bootstrap the contianers.